connectors/src/connectors/webcrawler/temporal/workflows.ts

```diff
@@ -0,0 +1,19 @@
+import { ModelId } from "@dust-tt/types";
+import { proxyActivities } from "@temporalio/workflow";
+
+import type * as activities from "@connectors/connectors/webcrawler/temporal/activities";
+
+const { crawlWebsiteByConnectorId } = proxyActivities<typeof activities>({
+  startToCloseTimeout: "120 minutes",
+  heartbeatTimeout: "120 seconds",
+});
+
+export async function crawlWebsiteWorkflow(
+  connectorId: ModelId
+): Promise<void> {
+  await crawlWebsiteByConnectorId(connectorId);
```

Comment:
So this is initial sync only right? ooc what will be the plan for keeping them in sync?

Reply:
Yes this is initial sync only for now. We'll need  a cron base temporal workflow (like [here](https://github.com/dust-tt/dust/blob/main/connectors/src/connectors/google_drive/temporal/client.ts#L160)) for subsequent sync.